packages/react/src/components/Button/Button.mdx

```diff
@@ -26,39 +32,224 @@ import { Button } from '@digdir/design-system-react';
+
+<Canvas of={ButtonStories.Lasting} />
+
+I eksempelet over har vi brukt `aria-disabled` til å deaktivere knappene. Dette er den anbefalte metoden for å vise at en knapp er deaktivert, da det gjør at den fremdeles kan fokuseres på, f.eks. ved navigasjon med tastatur. Slik vil blant annet skjermlesere og andre hjelpemidler bli informert om at knappen finnes, men ikke er aktiv.
+
+Denne egenskapen stopper ikke automatisk knappen fra å utløse `onCLick`.
+
+Pass på at callback-funksjonen ikke kjører når knappen er deaktivert.
+
+<br />
+
+## Retningslinjer for button
+
+<br />
+### En knapp skal utføre kun én handling
+
+Vi skal tydelig beskrive handlingene knappene utfører. Vi skal ikke ha flere handlinger i samme knapp.
+
+<Stack direction="row" style={{
+  justifyContent: 'unset',
+  alignItems: 'unset'
+}}>
+  <Do
+    description='En knapp skal kun gjøre én handling'
+    image='/img/component-guidelines/button/button-ex2-do.png'
+  />
+
+  <Dont
+    description='Flere handlinger på samme knapp bør unngås'
+    image='/img/component-guidelines/button/button-ex2-dont.png'
+  />
+</Stack>
+<br />
+
+### Èn hovedhandling per side
+
+Vi skal holde oss til en primærknapp per side.
+
+<Stack direction="row" style={{
+  justifyContent: 'unset',
+  alignItems: 'unset'
+}}>
+  <Do
+    description='Det skal kun være èn primærknapp per side'
+    image='/img/component-guidelines/button/button-ex3-do.png'
+  />
+
+  <Dont
+    description='Vi skal ikke ha flere primærknapper på samme side'
+    image='/img/component-guidelines/button/button-ex3-dont.png'
+  />
+</Stack>
+<br />
+
+### Bruk samme farge på knappene i en gruppe
+
+Vi skal holde oss til samme fargevariant for en gruppe knapper.
+
+<Stack direction="row" style={{
+  justifyContent: 'unset',
+  alignItems: 'unset'
+}}>
+  <Do
+    description='Knappene i en gruppe skal ha samme farge'
+    image='/img/component-guidelines/button/button-ex4-do.png'
+  />
+
+  <Dont
+    description='Ulike farger i en gruppe kan være forvirrende'
+    image='/img/component-guidelines/button/button-ex4-dont.png'
+  />
+</Stack>
+<br />
+
+### Ett ikon per knapp
+
+Vi skal holde oss til ett ikon per knapp.
+
+<Stack direction="row" style={{
+  justifyContent: 'left',
+  alignItems: 'left'
+}}>
+  <Do
+    description='Bruk kun et ikon'
+    image='/img/component-guidelines/button/button-ex1-do.png'
+  />
 
-### Bruk av Spinner til å lage en knapp som laster
+  <Dont
+    description='Ikke bruk flere ikoner i samme knapp'
+    image='/img/component-guidelines/button/button-ex1-dont.png'
+  />
+</Stack>
+<br />
 
-Dersom du ønsker å vise brukeren at knappen laster noe, kan du gjøre det ved å kombinere den med en spinner.
+<br />
 
-**OBS**: I dette eksempelet er det brukt `aria-disabled` til å deaktivere knappene.
-Dette er den anbefalte metoden for å vise at en knapp er deaktivert, da det gjør at den fremdeles kan fokuseres på, f.eks. ved navigasjon med tastatur.
```

Comment:
Dette er den anbefalte måten for å vise at en knapp er deaktivert.  Det gjør at den fremdeles kan få fokus når noen navigerer med tastaturet.